src/laboratory/api/serializers.py

```diff
@@ -598,3 +599,264 @@ class InstrumentalFamilyDataTableSerializer(serializers.Serializer):
 
 class PrecursorSerializer(serializers.Serializer):
     pk = serializers.IntegerField()
+
+class ValidateReactiveSerializer(serializers.ModelSerializer):
```

Comment:
ValidateReactiveSerializer class is not an required element of this pull request.